lib/kino/frame.ex

```diff
@@ -64,11 +64,34 @@ defmodule Kino.Frame do
       option is useful when updating frame in response to client
       events, such as form submission
 
+    * `:history` - if the update will be part of the frame history.
```

Comment:
I'm leaning towards `:transient`, wdyt?

Reply:
+1 for `:transient`

Reply:
I like `:history` for simplicity. If folks dislike it, `:temporary` perhaps? `:transient` is too fancy. :D

Reply:
I just think `:history` doesn't read as well, especially with `clear`. `:temporary` also works for me :)